sql/SchemaChanges/00000005.sql

```diff
@@ -0,0 +1,22 @@
+-- Modify timestamp column so they are not updated automatically
+
+DROP PROCEDURE IF EXISTS UpdateTo4_5 //
+CREATE PROCEDURE UpdateTo4_5()
+BEGIN
+	IF EXISTS (SELECT 1 FROM system_flags WHERE major_version=1 AND minor_version=4) THEN
+		UPDATE system_flags SET minor_version=5;
+		ALTER TABLE users              CHANGE COLUMN created     created     TIMESTAMP DEFAULT NOW();
+		ALTER TABLE spaces             CHANGE COLUMN created     created     TIMESTAMP DEFAULT NOW();
+		ALTER TABLE benchmarks         CHANGE COLUMN uploaded    uploaded    TIMESTAMP DEFAULT NOW();
+		ALTER TABLE solvers            CHANGE COLUMN uploaded    uploaded    TIMESTAMP DEFAULT NOW();
+		ALTER TABLE solver_pipelines   CHANGE COLUMN uploaded    uploaded    TIMESTAMP DEFAULT NOW();
+--		ALTER TABLE jobs               CHANGE COLUMN completed   completed   TIMESTAMP DEFAULT NOW();
```

Comment:
https://github.com/StarExec/StarExec/blob/2ec9039073944995fc87d70b5b35194a587707bf/sql/new-install/StarSchema.sql#L290-L291

Since `jobs` has two columns with the `TIMESTAMP` type, this is kind of a problem. 🤷‍♂️